Makefile

```diff
@@ -74,6 +74,7 @@ node_modules/.installed : src/js/package.json src/js/package-lock.json
 
 dist/pyodide.js src/js/_pyodide.out.js: src/js/*.ts src/js/pyproxy.gen.ts src/js/error_handling.gen.ts node_modules/.installed
 	npx rollup -c src/js/rollup.config.js
+	sed -i 's!await import(!await import(/* webpackIgnore: true */ !g' dist/pyodide.js
```

Comment:
Can you add a comment about what it does, as it's not entirely obvious from the regex, particularly given that '!' is used as a separator from what I understand?